app/tests/unit/components/Table/SortableHeader.test.tsx

```diff
@@ -1,9 +1,9 @@
-import { useRouter } from "next/router";
 import { mocked } from "jest-mock";
 import React from "react";
 import SortableHeader from "components/Table/SortableHeader";
 import { render, screen } from "@testing-library/react";
-jest.mock("next/router");
+
+const useRouter = jest.spyOn(require("next/router"), "useRouter");
```

Comment:
I'm curious what error you were getting that was requiring this syntax, we prefer the `jest.mock` variant where we can

Reply:
Sorry for not being descriptive in the commit message.
I was getting this error:
![Screenshot 2023-03-16 at 12 51 15 PM](https://user-images.githubusercontent.com/52381975/225736975-0046b108-f625-48b0-98c8-a65dd4e63b3d.png)
and the workaround was here: https://github.com/vercel/next.js/discussions/13891

Reply:
I've found [this](https://github.com/vercel/next.js/issues/7479#issuecomment-533657701) which helps keep the `jest.mock`.

Reply:
Makes sense! Either way it works.